install/amazon_rds/_04_setting_env_vars.mdx

```diff
@@ -4,11 +4,13 @@ import CollectorDotEnvSingle from './_04_setting_env_vars_single.mdx';
 import CollectorDotEnvMulti from './_04_setting_env_vars_multi.mdx';
 import CollectorDotEnvSingleECS from './_04_setting_env_vars_single_ecs.mdx';
 import CollectorDotEnvMultiECS from './_04_setting_env_vars_multi_ecs.mdx';
+import CollectorDotEnvSingleEKS from './_04_setting_env_vars_single_eks.mdx';
+import CollectorDotEnvMultiEKS from './_04_setting_env_vars_multi_eks.mdx';
 
 export const SettingEnvVarsDocker = ({ apiKey }) => {
   const tabs = [
-    [ 'single', 'Single Server' ],
-    [ 'multiple', 'Multiple Servers'],
+    [ 'single', 'Single DB Instance' ],
+    [ 'multiple', 'Multiple DB Instances'],
```

Comment:
Lukas and I chatted about this a bit, Lukas pointed out that "Single Server" might be misleading as AWS doesn't really say "Postgres server", they instead say "DB instance".
People could read "Single Server" as "single collector server", so we wanted to update the terminology here.

I actually went ahead and changed all "server" word from `install/amazon_rds` dir to "instance".